### ISSUE TITLE
Clear Examples

### ISSUE THREAD

**Issue by sinnis-dev-trey** [siteorigin/docs#66]
Hey,

Page in question: https://siteorigin.com/docs/widgets-bundle/getting-started/creating-a-widget/

Where does one add all the code? It literally just has a file tree and code dumps ... there doesn't seem to really be anything clear to say where code snippet A goes etc.?

**Comment by Misplon** [comment]
Hi there. The code would go in your custom theme or child theme or custom
plugin.

**Comment by sinnis-dev-trey** [comment]
@Misplon  but where? What file - I made a custom plugin and put the code in the base file for that plugin and it doesn't work .. it's much clearer to actually specify where the code lives in terms of the file tree rather than posting code and saying put it in your theme. A theme can have an infinite number of files .. how does one narrow done vagueness?

**Comment by Misplon** [comment]
The code should be added to your theme or plugin where you think it's most appropriate. Folder structures vary depending on the project and or developer.

Please, find two example plugins [here](https://github.com/siteorigin/so-dev-examples). For example, once 'extend-widgets-bundle' plugin is activated, you can head over to Plugins > SiteOrigin Widgets, activate the Hello World widget and then go to a page, open Page Builder and insert it.

Hope the examples help :)

**Comment by sinnis-dev-trey** [comment]
@Misplon  this is why WordPress sucks ,.. but thank you for the link, tis most helpful :)

**Comment by sinnis-dev-trey** [comment]
New issue - I downloaded the so-dev-examples-develop folder and dumped it into my plugins/ folder -- Went into the WP admin panel and can't find the hello, world widget?

I saw this [here](https://siteorigin.com/docs/widgets-bundle/getting-started/creating-a-widget/):

> Once you've done this, you'll see your widget in the Plugins > SiteOrigin Widgets list, it can be activated and deactivated, and you'll see an 'Untitled Widget' in Page Builder widgets and a blank widget in other widget lists. So you can't really use your widget yet, but it's there! 

However, I don't have a SiteOrigin Widgets list menu link. I've checked the plugins main page and the SO plugin is activated. The README doesn't indicate there are more steps to do.. what am I missing?
